utils/install_v2.sh

```diff
@@ -368,33 +391,60 @@ cleanup() {
 install() {
 	local dir=$1
 	shift
+	# Determine the source directory based on tarball structure
+	# Check for actual content directories, not just the parent directory
+	local src_dir=""
+	if [ -d "$TMP_DIR/$dir/bin" ] || [ -d "$TMP_DIR/$dir/lib" ] || [ -d "$TMP_DIR/$dir/lib64" ] || [ -d "$TMP_DIR/$dir/include" ]; then
+		# Old structure (pre-0.15.0): WasmEdge-VERSION-OS directory contains the actual files
+		src_dir="$TMP_DIR/$dir"
+		info "Detected old tarball structure (pre-0.15.0)"
+	else
+		# New structure (0.15.0+): files extracted directly to TMP_DIR
+		src_dir="$TMP_DIR"
+		info "Detected new tarball structure (0.15.0+)"
+	fi
```

Comment:
Hi @SAtacker,  
I changed the logic for detecting 0.15.0 and pre-0.15.0. The only difference between them is the file structure. Therefore, I think it can be simplified by using structure detection instead of version detection. If you prefer to use version checks instead, I can rewrite this part.

Reply:
You can find `# Test version 0.15.0 with new tarball structure` in the workflow file. It's not very clear because I used `--version=${TEST_228_VERSION}`, where `TEST_228_VERSION` equals `0.15.0`. Perhaps the name is not very clear. Do you want me to add a new constant for `0.15.0`?

Reply:
No, it's alright. Thanks for letting me know. LGTM!